### PR TITLE
feat: docs to update ENV values in helm installation

### PR DESCRIPTION
**Pull request by sum35h**
Added docs for Instance configuration> Helm installation

### CODE REVIEW

**Comment by sum35h** [comment]
@sharat87  would it be safe to remove the old K8s instructions in the [configuration docs ](https://appsmith-docs-git-706-docs-update-existing-fb1b9b-get-appsmith.vercel.app/getting-started/setup/instance-configuration#configuring-kubernetes-installations)

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
+
+Remove # before the variables to ensure they're not commented
+
+```bash
+// Example variable configuration format
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:21]: Can we remove these comments from the code blocks. On the rendered documentation, these code blocks show up with a `copy` button. It is expected that people can just hit that button to copy the code, and paste in their terminal.

Having this comment _inside_ the code block means that they also copy this comment. So, can we either move this comment out into the previous sentence, or just remove it please?

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+After making any changes, **remember to restart the docker containers** for the changes to take effect.
+
+```bash
+// To restart Appsmith using docker
+docker restart appsmith
+
+// To restart Appsmith using docker compose
+docker-compose restart appsmith
+```
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:33]: Similar problem as above. Let's remove comments, _and_ split this into two code blocks, with the comment as just ordinary text, so it's easy to copy just any one command.

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+After making any changes, **remember to restart the pods** for the changes to take effect
+
+```
+// commands to restart k8s pods
+kubectl apply -f appsmith-configmap.yaml
+kubectl scale deployment appsmith-internal-server --replicas=0
+kubectl scale deployment appsmith-internal-server --replicas=1
+```
+
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:54]: Yes, let's remove this.

```suggestion
```

```

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  <TabItem label="Community Edition" value="communityEdition"> 
+
+  You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen.
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:69]: Ensure your code blocks and content line up to the bullet point's content, so they all show up aligned in the rendered website. In this case, we currently have this:

Screenshot 2022-12-29 at 10 14 13 AM


If we align to the content of the bullet point, which is `When using...`:
Screenshot 2022-12-29 at 10 15 40 AM

Then we get this:

Screenshot 2022-12-29 at 10 16 51 AM

See the difference? Can we fix this for all the other code blocks under bullet points as well please?

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+To configure an ECS installation, follow these steps:
+
+1. Navigate to the **ECS console** and select **Task Definitions** on the sidebar.
+2. Click on the Task Definition used by your ECS instance, and hit **Create new revision**.
+3. On the `Task Definition config` page, click the Appsmith **container definition** to edit it. In the **Environment Section**, enter the environment configuration as **key-value pairs** as shown below.
+
+![](/img/ecs-task-env_(1).png)
+
+It's recommended to use **AWS Secrets** for sensitive information, please follow the steps to [create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage\_create-basic-secret.html). Use the **ARN** of the secret as the Environment value and set the option to **ValueFrom**.
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:127]: I don't think we need to escape this anymore. We're not on Gitbook anymore, and Docusaurus' Markdown parser is quite smart.

```suggestion
It's recommended to use **AWS Secrets** for sensitive information, please follow the steps to [create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html). Use the **ARN** of the secret as the Environment value and set the option to **ValueFrom**.
```

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
+
+Remove # before the variables to ensure they're not commented
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:18]:
```suggestion
Remove `#` before the variables to ensure they're not commented.
```

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:16]:
```suggestion
To configure a docker installation, go to your installation folder and edit the `stacks/configuration/docker.env` file with the environment variables for the service.
```

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+```
+Once you have made changes to the values file, run the below command.
+
+```bash
+  helm upgrade --values values.yaml appsmith appsmith-ee/appsmith
+```
+  </TabItem>
+</Tabs> 
+
+## Configure ECS installations
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:117]: This section has too much bold text. We've used bold text in above sections, but only for text we absolutely don't want our users to miss, and grab their attention towards it.

But in this section we seem to be using it for... several things. Emphasis on too many things, gives emphasis to nothing. Can we reduce the bold here please?

**Review by sharat87 — CHANGES_REQUESTED** [review]
Content looks good. Suggested some formatting improvements, but otherwise looks fine.

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 4 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:49]: Got it.

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 8 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:97]: Which parts need emphasis? I can make the changes.

Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:97]: @sharat87 - I can do the above minor changes as part of this PR. On another note, we'll have to rewrite this page as for now it just focuses on Configuring Environment variables. However, it's more of a landing page that will give users an idea on:
- Introduction for Instance Configuration
- Things to be expected like custom domain, email config and more
- Configuring Environment variables for different installation types
- Troubleshooting section
- Further reading.
Do let me know your thoughts? I'll raise another ticket to look into this.

Comment (sharat87) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:97]: Oh hey, sorry I thought this was all written by Sumesh, saw his name on the PR, so my comments were targeted towards him, didn't know you're working on this. 🙂

Reg. your question, honestly, I don't think anything in this section needs an emphasis. It's just fine being plain, but I'm not too strong on this. If you see something that should be bold, please go ahead. 🙂

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, go to your installation folder and edit the `stacks/configuration/docker.env` file with the environment variables for the service.
+
+Remove `#` before the variables to ensure they're not commented.
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:18]: @sharat87 - Do we need this line? Can't we add parameters without `#`?

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 8 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:97]: Sumesh has only added the configured Helm installations section and passed it on to me for rewriting. The others are not yet worked on. But it makes sense to fix the things as part of this PR as are no huge changes. 
I'll get the changes done, and we can merge this PR. We can discuss if the page needs restructuring and if needed can do a separate PR to accommodate that.

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 9 above)
Comment (sharat87) [reply website/docs/getting-started/setup/instance-configuration/README.mdx]: Not strictly. Can be removed.

**Comment by jnikhila** [comment]
@sharat87 - Back in review. Please approve.

**Review by jnikhila — APPROVED** [review]

**Comment by jnikhila** [comment]
> Content looks good. Suggested some formatting improvements, but otherwise looks fine.

All changes are done. Please review

**Pull request merged**